[2022-08-02 04:24:28,273] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-02T04:24:26.469276+00:00 [queued]>
[2022-08-02 04:24:28,294] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-02T04:24:26.469276+00:00 [queued]>
[2022-08-02 04:24:28,294] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 04:24:28,295] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-08-02 04:24:28,295] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 04:24:28,309] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-08-02 04:24:26.469276+00:00
[2022-08-02 04:24:28,315] {standard_task_runner.py:52} INFO - Started process 93 to run task
[2022-08-02 04:24:28,321] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-08-02T04:24:26.469276+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp30k2eefo', '--error-file', '/tmp/tmpgvadeu86']
[2022-08-02 04:24:28,323] {standard_task_runner.py:77} INFO - Job 91: Subtask download_dataset_task
[2022-08-02 04:24:28,411] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-02T04:24:26.469276+00:00 [running]> on host ce750bf29e22
[2022-08-02 04:24:28,493] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T04:24:26.469276+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T04:24:26.469276+00:00
[2022-08-02 04:24:28,495] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-02 04:24:28,497] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://storage.cloud.google.com/dtc_data_lake_de-bootcamp-357803/covid-cases.csv > /opt/***/covid-cases.csv']
[2022-08-02 04:24:28,517] {subprocess.py:85} INFO - Output:
[2022-08-02 04:24:28,695] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-02 04:24:28,749] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220802T042426, start_date=20220802T042428, end_date=20220802T042428
[2022-08-02 04:24:28,813] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-02 04:24:28,900] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
